workers/ohsome_quality_analyst/indicators/mapping_saturation/indicator.py

```diff
@@ -46,11 +46,16 @@ async def preprocess(self) -> bool:
         )
         results = [y_dict["value"] for y_dict in query_results["result"]]
         timestamps = [y_dict["timestamp"] for y_dict in query_results["result"]]
+
+        # osm-timestamp is only the timestamp of the latest feature
```

Comment:
I would move the comment to the variable declaration. And remove some emty line here.